### PR TITLE
RHDEVDOCS-3411- added release notes for 1.3.0

### PR DESCRIPTION
**Pull request by DebarghoGhosh**
Aligned team: Dev Tools
OCP version for cherry-picking: enterprise-4.8, enterprise-4.9
JIRA issues: [RHDEVDOCS-3411](https://issues.redhat.com/browse/RHDEVDOCS-3411) GitOps Release Notes, Known Issues and Bug Fixes for 1.3
 https://issues.redhat.com/browse/RHDEVDOCS-3411
Preview pages: https://deploy-preview-37968--osdocs.netlify.app/openshift-enterprise/latest/cicd/gitops/gitops-release-notes#gitops-release-notes-1-3_gitops-release-notes
SME+QE review: @reginapizza 
Peer-review:  @Preeticp

### CODE REVIEW

**Review by reginaelyse — CHANGES_REQUESTED** [review]
Left a few comments!

**Review by reginaelyse — APPROVED** [review]
Awesome thanks! LGTM now :)

File: modules/gitops-release-notes-1-3-0.adoc
Comment (sbose78) [inline modules/gitops-release-notes-1-3-0.adoc:68]: When I view in rich text, this doesn't feel indented under the second bullet point, could you please verify?

File: modules/gitops-release-notes-1-3-0.adoc
Comment (sbose78) [inline modules/gitops-release-notes-1-3-0.adoc:35]: -1 on this :)

I now we've had this in previous release notes, but I would call this "developer console integration" instead of Red Hat OpenShift Gitops Service. Please consult with @siamaksade and @wtam2018 :) 

image

Comment (wtam2018) [reply modules/gitops-release-notes-1-3-0.adoc:35]: We has dropped "Red Hat OpenShift GitOps Service" from this table as this service is an internal service.

File: modules/gitops-release-notes-1-3-0.adoc
Comment (Preeticp) [reply modules/gitops-release-notes-1-3-0.adoc:68]: @DebarghoGhosh adding a '+' above the [discrete] will solve the indentation issue. 
But more importantly, we should not have procedures in new features, these should be just key highlights in bulleted points. I would suggest moving this procedure to an appropriate assembly in the GitOps section.
However, please note, procedures as an exception can be seen in the Known Issues section.

File: modules/gitops-release-notes-1-3-0.adoc
Comment (Preeticp) [inline modules/gitops-release-notes-1-3-0.adoc:114]: From the UI perspective we do not have something called the DevConsole. We only have a **Developer** perspective in the console. So in the docs mention the latter.
'directy from the DevConsole using the improved environment details view in DevConsole for OCP 4.9.' seems to be repeating the console mentions.
Suggest: Use the improved environment details view in the **Developer** perspective of the console 4.9 to gain insights into the GitOps environments./Gain insights into the GitOps environments using the improved environment details view in  the **Developer** perspective of the console 4.9.

File: modules/gitops-release-notes-1-3-0.adoc
Comment (Preeticp) [inline modules/gitops-release-notes-1-3-0.adoc:115]: You can now access

File: modules/gitops-release-notes-1-3-0.adoc
Comment (Preeticp) [inline modules/gitops-release-notes-1-3-0.adoc:116]: The GitOps Operator now conforms to the
There are two by..., by..., in the sentence. You can drop 'by introducing the following changes' and add colon after SDK.
sdk/SDK when in doubt about a usage, do a quick search in the existing docs.

File: modules/gitops-release-notes-1-3-0.adoc
Comment (Preeticp) [inline modules/gitops-release-notes-1-3-0.adoc:118]: Suggest replacing 'will be' to 'is'

File: modules/gitops-release-notes-1-3-0.adoc
Comment (Preeticp) [inline modules/gitops-release-notes-1-3-0.adoc:125]: “Out Of Sync” remove quotes, use bold format instead.
Operator, search and correct all instances in the PR, please.

File: modules/gitops-release-notes-1-3-0.adoc
Comment (Preeticp) [inline modules/gitops-release-notes-1-3-0.adoc:131]: Dev/Dex
the Dex authentication provider to the Keycloak provider

File: modules/gitops-release-notes-1-3-0.adoc
Comment (Preeticp) [inline modules/gitops-release-notes-1-3-0.adoc:139]: When a user skims the KIs the separate sections give the impression that there are three KIs but actually there is only one. Suggest making the prevention and workaround as paragraphs. Also there is only one workaround so Workarounds does not sound right.
Suggest:
As a prevention, when migrating, uninstall Dex by removing....
As a workaround, If you run into the above issue, uninstall...

File: modules/gitops-release-notes-1-3-0.adoc
Comment (Preeticp) [inline modules/gitops-release-notes-1-3-0.adoc:129]: Since there is only one KI suggest dropping this sentence. WDYT?

**Review by Preeticp — COMMENTED** [review]
@DebarghoGhosh overall looks good. Have left a few suggestions, PTAL.

File: modules/gitops-release-notes-1-3-0.adoc
Comment (DebarghoGhosh) [reply modules/gitops-release-notes-1-3-0.adoc:68]: @Preeticp as per discussion, removing the procedure from the New Features section and we can add it in the following page:

https://deploy-preview-37968--osdocs.netlify.app/openshift-enterprise/latest/cicd/gitops/configuring_argo_cd_to_recursively_sync_a_git_repository_with_your_application/configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.html

@reginapizza  please check and confirm if we can add the procedure in the above page

Comment (reginapizza) [reply modules/gitops-release-notes-1-3-0.adoc]: that's fine with me, I agree with that decision if procedures shouldn't be in new features sections

File: modules/logging-in-to-the-argo-cd-instance-by-using-your-openshift-credentials.adoc
Comment (Preeticp) [inline modules/logging-in-to-the-argo-cd-instance-by-using-your-openshift-credentials.adoc:26]: This needs to be a different module.
Maybe titled something like: Running the Argo CD instance at the cluster-level
@reginapizza can you confirm that this module helps the Argo CD instance to run at the cluster level instead of the namespace level?

File: modules/go-compatibility-and-support-matrix.adoc
Comment (Preeticp) [inline modules/go-compatibility-and-support-matrix.adoc:7]: Not sure if we need the extra + at the end of the sentence. Strangely, I do not see it in the rendered version

File: modules/logging-in-to-the-argo-cd-instance-by-using-your-openshift-credentials.adoc
Comment (Preeticp) [inline modules/logging-in-to-the-argo-cd-instance-by-using-your-openshift-credentials.adoc:44]: the `runOnInfra` toggle
these nodes - plural

File: modules/logging-in-to-the-argo-cd-instance-by-using-your-openshift-credentials.adoc
Comment (Preeticp) [inline modules/logging-in-to-the-argo-cd-instance-by-using-your-openshift-credentials.adoc:69]: Please use the correct formatting as per the OCP docs guidelines for: `Pods -> Pod details`

File: modules/logging-in-to-the-argo-cd-instance-by-using-your-openshift-credentials.adoc
Comment (Preeticp) [inline modules/logging-in-to-the-argo-cd-instance-by-using-your-openshift-credentials.adoc:55]: Since adding taints is optional, I think this step is optional too. @reginapizza can you confirm?
Also, suggest starting the sentence with - If taints have been added to the nodes, then add `tolerations`...
This helps users to decide if it is worth reading the next part of the sentence. They will read it only if they have used taints.
Also, in short can we mention what these tolerations do like how taints are used to isolate workloads?

File: modules/logging-in-to-the-argo-cd-instance-by-using-your-openshift-credentials.adoc
Comment (Preeticp) [inline modules/logging-in-to-the-argo-cd-instance-by-using-your-openshift-credentials.adoc:73]: Any manually added `nodeSelectors` and `tolerations` added to the default/Any `nodeSelectors` and `tolerations` manually added to the default
by toggle and/by the toggle and the `tolerations`

**Review by Preeticp — COMMENTED** [review]
@DebarghoGhosh I have left a few suggestions, PTAL

File: modules/logging-in-to-the-argo-cd-instance-by-using-your-openshift-credentials.adoc
Comment (DebarghoGhosh) [reply modules/logging-in-to-the-argo-cd-instance-by-using-your-openshift-credentials.adoc:55]: @reginapizza please suggest

Comment (reginapizza) [reply modules/logging-in-to-the-argo-cd-instance-by-using-your-openshift-credentials.adoc:55]: @saumeya can you provide input? I believe this step is optional as well but I'm not positive

**Comment by siamaksade** [comment]
Would it make sense to limit the support matrix table to supported releases (1.1+)?  Also do we need to include the patch releases?

File: modules/logging-in-to-the-argo-cd-instance-by-using-your-openshift-credentials.adoc
Comment (saumeya) [reply modules/logging-in-to-the-argo-cd-instance-by-using-your-openshift-credentials.adoc:55]: @Preeticp yes that step is optional, we can add your suggestion, i'll try adding a brief about tolerations and taints, there is a jira about already assigned to me, i'll work on it.

File: modules/logging-in-to-the-argo-cd-instance-by-using-your-openshift-credentials.adoc
Comment (saumeya) [reply modules/logging-in-to-the-argo-cd-instance-by-using-your-openshift-credentials.adoc:73]: I didn't understand this, could you elaborate?

Comment (jaideepr97) [reply modules/logging-in-to-the-argo-cd-instance-by-using-your-openshift-credentials.adoc:73]: Maybe this can be rephrased this way?:

Any `nodeSelectors` and `tolerations` manually added to the default Argo CD custom resource will be superseded by the toggle and `tolerations` present in the `GitOpsService` custom resource

File: modules/gitops-release-notes-1-3-0.adoc
Comment (Preeticp) [inline modules/gitops-release-notes-1-3-0.adoc:23]: I suggest we drop the table at module level as was done for the Pipeline release notes PR. The information is repeated, and this would reduce maintenance as well as potential errors.

File: modules/go-run-argo-cd-instance-on-infrastructure-nodes.adoc
Comment (Preeticp) [inline modules/go-run-argo-cd-instance-on-infrastructure-nodes.adoc:3]: Replace above with: gitops/configuring_argo_cd_to_recursively_sync_a_git_repository_with_your_application/configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.adoc

File: modules/go-run-argo-cd-instance-on-infrastructure-nodes.adoc
Comment (Preeticp) [inline modules/go-run-argo-cd-instance-on-infrastructure-nodes.adoc:38]: custom resource, for example: 
Remove the extra 'if taints have been added to the nodes'

File: modules/go-run-argo-cd-instance-on-infrastructure-nodes.adoc
Comment (Preeticp) [inline modules/go-run-argo-cd-instance-on-infrastructure-nodes.adoc:9]: Use the attribute instead of: OpenShift GitOps

File: modules/gitops-release-notes-1-3-0.adoc
Comment (Preeticp) [inline modules/gitops-release-notes-1-3-0.adoc:56]: You can now access custom health checks
Route/`Route`

File: modules/gitops-release-notes-1-3-0.adoc
Comment (Preeticp) [inline modules/gitops-release-notes-1-3-0.adoc:66]: Previously, if you set up a new namespace

File: modules/logging-in-to-the-argo-cd-instance-by-using-your-openshift-credentials.adoc
Comment (Preeticp) [reply modules/logging-in-to-the-argo-cd-instance-by-using-your-openshift-credentials.adoc]: @saumeya @jaideepr97 can one of you confirm that this procedure allows the user to run the default Argo CD instance on a cluster, so that we can title the section appropriately? Currently this is what it says, please let us know if there are any concerns with this title.

File: modules/go-run-argo-cd-instance-on-infrastructure-nodes.adoc
Comment (Preeticp) [inline modules/go-run-argo-cd-instance-on-infrastructure-nodes.adoc:9]: by the {gitops-title} Operator as well as the accompanying

File: modules/gitops-release-notes-1-3-0.adoc
Comment (Preeticp) [inline modules/gitops-release-notes-1-3-0.adoc:19]: Please remove 'to' from: You can now access to custom health checks

Comment (Preeticp) [reply modules/gitops-release-notes-1-3-0.adoc:19]: @saumeya @jaideepr97 @reginapizza the only mention of OLM-Operator is here:https://docs.openshift.com/container-platform/4.9/security/certificate_types_descriptions/olm-certificates.html
My question hence is should this be written as `olm-operator`, and if it should be singular or plural as in operator/operators.
Also in this context is `Route` a custom resource?
I wonder if this should be written as:
* You can now access custom health checks in Argo CD for `DeploymentConfig` resources, routes, and `OLM-Operators`.
@DebarghoGhosh  please change depending on answers to the above question.

File: modules/go-compatibility-and-support-matrix.adoc
Comment (Preeticp) [inline modules/go-compatibility-and-support-matrix.adoc:24]: @DebarghoGhosh as Siamak mentioned we can drop the x.y.z release mentions in the table. So we can remove entries for 1.1.1, 1.1.2, 1.2.1, and 1.3.1

File: modules/go-run-argo-cd-instance-on-infrastructure-nodes.adoc
Comment (saumeya) [reply modules/go-run-argo-cd-instance-on-infrastructure-nodes.adoc:9]: yes @Preeticp your suggestion looks good.

File: modules/gitops-release-notes-1-3-0.adoc
Comment (DebarghoGhosh) [reply modules/gitops-release-notes-1-3-0.adoc:19]: @saumeya please suggest

Comment (DebarghoGhosh) [reply modules/gitops-release-notes-1-3-0.adoc:19]: @praveenkumar, @chetan-rns  please suggest

Comment (chetan-rns) [reply modules/gitops-release-notes-1-3-0.adoc:19]: I believe `olm-operator` has a different meaning since it is one of the OLM components. Here "OLM Operators" refers to the operators installed using OLM. And Route is an OpenShift custom resource.

Comment (Preeticp) [reply modules/gitops-release-notes-1-3-0.adoc:19]: Thank you, Chetan. In that case @DebarghoGhosh we simply need to replace `OLM` Operators with 'operators installed using OLM.'

Comment (Preeticp) [reply modules/gitops-release-notes-1-3-0.adoc]: Hey Debargho, @praveenkumar is our CRC rockstar :) We might need either Praveen Thangdancha or another QE from the team to provide an LGTM before we merge.

File: modules/gitops-release-notes-1-3-0.adoc
Comment (iam-veeramalla) [inline modules/gitops-release-notes-1-3-0.adoc:14]: This statement is not true. Please replace `Red Hat Single Sign on` with `Dex`. It should look something like this.

For a fresh install of v1.3.0, Dex is automatically configured. You can log into the default Argo CD instance in the `openshift-gitops` namespace using the OpenShift credentials. As an admin you can disable the Dex installation after the Operator is installed which will result in removing the Dex deployment from the `openshift-gitops` namespace.

File: modules/gitops-release-notes-1-3-0.adoc
Comment (iam-veeramalla) [inline modules/gitops-release-notes-1-3-0.adoc:15]: Please remove this line.

**Review by iam-veeramalla — CHANGES_REQUESTED** [review]

File: modules/gitops-release-notes-1-3-0.adoc
Comment (jaideepr97) [reply modules/gitops-release-notes-1-3-0.adoc:14]: We should mention that they can use `kubeadmin` credentials with dex as well, right?

Comment (DebarghoGhosh) [reply modules/gitops-release-notes-1-3-0.adoc]: @iam-veeramalla and @jaideepr97 I have made the changes. Please check and confirm. Need your LGTM before merging

**Review by sbose78 — APPROVED** [review]

**Comment by Preeticp** [comment]
@sbose78 thank you for approving the PR. 
/lgtm

**Pull request merged**
